app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">rick-and-morty</string>
+    <string name="title_activity_character">CharacterActivity</string>
```

Comment:
Where is this used?

Reply:
I thought using when I will create navigation bar, and move from one activity to another.